### ISSUE TITLE
Keyboard shortcut for search

### ISSUE THREAD

**Issue by ronilaukkarinen** [thelounge/thelounge#4535]
### Feature Description

It would be nice to be able to open search with <kbd>⌘</kbd>/<kbd>ctrl</kbd> + <kbd>F</kbd>. It's rather trivial to implement and I could look in to it some time if I have time but because I do not right now I'd rather open an issue first if anyone's up for it.

**Comment by brunnre8** [comment]
Not like this, we shouldn't overwrite such a commonly used browser shortcut.

To be honest, I think we shouldn't add more shortcuts without letting the user change those.

**Comment by ronilaukkarinen** [comment]
Yeah, perhaps something else than <kbd>⌘</kbd>/<kbd>ctrl</kbd> + <kbd>F</kbd>, good point. But a shortcut for search would still be a good thing imho.
